src/main/java/uk/gov/hmcts/reform/judicialapi/elinks/domain/BaseLocation.java

```diff
@@ -24,43 +24,43 @@ public class BaseLocation {
 
     @Id
     @Column(name = "base_location_Id")
-    @JsonProperty(value = "id")
+    @JsonProperty("id")
     private String baseLocationId;
 
     @Column(name = "name")
     @Size(max = 256)
-    @JsonProperty(value = "name")
+    @JsonProperty("name")
```

Comment:
minor: name could be omitted, JsonProperty will default to name of field defined in class (same applies for all field annotations below)